src/include/sof/audio/buffer.h

```diff
@@ -67,6 +68,7 @@ struct comp_buffer {
 	uint32_t pipeline_id;
 	uint32_t caps;
 	uint32_t core;
+	bool inter_core;	/* true if between comps on different cores */
```

Comment:
I'd rephrase the comment a little: "True if connected to a component from another core", to make it a bit clearer.

Reply:
Done.